.buildkite/steps/publish-docker-image-ecr.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+splitter_version="$(cat version/VERSION)"
+builder_name=$(docker buildx create --use)
+image_tag="445615400570.dkr.ecr.us-east-1.amazonaws.com/test-splitter:$splitter_version-$BUILD_NUMBER"
```

Comment:
Will this be the build number of the release pipeline or the build number of the test splitter pipeline? I think it should be the test splitter pipeline build number.

Reply:
It's the build number of the test splitter pipeline, we pass this when we trigger the release pipeline.